cmd/activator/main.go

```diff
@@ -213,6 +216,14 @@ func main() {
 	}
 	ah = reqLogHandler
 
+	ah = handler.NewTimeToFirstByteTimeoutHandler(ah, "activator request timeout", func(r *http.Request) time.Duration {
```

Comment:
Is this place chosen for a reason or arbitrary?

Reply:
No particular reason, we now moved it to be called after the logging and tracing handlers.